### PR TITLE
Update compatibility for temporary tables

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Update compatibility for temporary tables for dev and v5.0.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)  --> 需要对版本进行描述：从 v5.0 GA 版本起，
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/5673

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by bb7133** [comment]
Thanks for updating! Can we update "CREATE TABLE" page: https://docs.pingcap.com/zh/tidb/stable/sql-statement-create-table#mysql-%E5%85%BC%E5%AE%B9%E6%80%A7 as well?

**Comment by TomShawn** [comment]
> @TomShawn Thanks for updating! Could you update "CREATE TABLE" page: https://docs.pingcap.com/zh/tidb/stable/sql-statement-create-table#mysql-%E5%85%BC%E5%AE%B9%E6%80%A7 as well?

What I update is exactly sql-statement-create-table.md. Do you mean the v4.0 version of it?
image

File: sql-statements/sql-statement-create-table.md [hunk 1]
```diff
@@ -181,7 +181,7 @@ mysql> DESC t1;
 
 ## MySQL 兼容性
 
-* TiDB 不支持临时表。如果 [`tidb_enable_noop_functions = 0`](/system-variables.md#tidb_enable_noop_functions-从-v40-版本开始引入) ，执行 `CREATE TEMPORARY TABLE` 语法会报错。如果 `tidb_enable_noop_functions = 1`，TiDB 会忽略 `TEMPORARY` 关键字。
+* TiDB 不支持临时表。自 v5.0 GA 版本起，如果 [`tidb_enable_noop_functions = 0`](/system-variables.md#tidb_enable_noop_functions-从-v40-版本开始引入) ，执行 `CREATE TEMPORARY TABLE` 语法会报错；如果 `tidb_enable_noop_functions = 1`，TiDB 会忽略 `TEMPORARY` 关键字。
```
Comment (TomShawn) [inline sql-statements/sql-statement-create-table.md:184]:
```suggestion
* TiDB 不支持临时表。如果 [`tidb_enable_noop_functions = 0`](/system-variables.md#tidb_enable_noop_functions-从-v40-版本开始引入) ，执行 `CREATE TEMPORARY TABLE` 语法会报错；如果 `tidb_enable_noop_functions = 1`，TiDB 会忽略 `TEMPORARY` 关键字。
```

**Comment by bb7133** [comment]
> > @TomShawn Thanks for updating! Could you update "CREATE TABLE" page: https://docs.pingcap.com/zh/tidb/stable/sql-statement-create-table#mysql-%E5%85%BC%E5%AE%B9%E6%80%A7 as well?
> 
> What I update is exactly sql-statement-create-table.md. Do you mean the v4.0 version of it?
> image

Sorry I made a mistake.

**Comment by TomShawn** [comment]
@yikeke PTAL

**Review by yikeke — APPROVED** [review]

**Comment by Joyinqin** [comment]
translation/done

**Pull request merged**